minikube-deploy.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Any reason to force bash here?  Script seems simple enough to just use `#!/bin/sh`